client/src/components/Header/Header.jsx

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
-
+import AppBar from '@material-ui/core/AppBar';
+import styles from './Header.module.css';
+//import Typography from '@material-ui/core/Typography';
 // TODO make header component
 
 const Header = () => {
-  return <></>;
+  return (
+    <AppBar className={styles['header-bar']}>
+      <div className={styles['header-title']}>
```

Comment:
instead of a `div`, use `Box` with the `flex` attributes: https://material-ui.com/components/box/